src/fastapi_app/rag_advanced.py

```diff
@@ -58,7 +58,7 @@ async def run(
         query_messages = build_messages(
             model=self.chat_model,
             system_prompt=self.query_prompt_template,
-            new_user_message=original_user_query,
+            new_user_content=original_user_query,
```

Comment:
Thanks! FYI that I'm going to pin the versions in my upcoming change, and use dependabot for updates, but we need this change regardless.

Reply:
Got you